pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -167,6 +182,60 @@ private String getDefaultBrokerId() {
     }
   }
 
+  @Override
+  public Map<Long, String> getRunningQueries() {
+    return _queriesById.entrySet().stream().collect(Collectors.toMap(Map.Entry::getKey, e -> e.getValue()._query));
```

Comment:
Check and throw exception when query cancellation is not enabled (with proper exception message). Same for other APIs that require enabling query cancellation. We don't want to return empty map because that can confuse the user.